models/prices/solana/prices_solana_tokens.sql

```diff
@@ -11,41 +11,660 @@ SELECT
+        ('dogo-dogemongo','solana','DOGO','5LSFpvLDkcdV2a3Kiyzmg5YmJsj2XDLySaXvnfP1cgLT',6),
+        ('pld-plutonian-dao','solana','PLD','2cJgFtnqjaoiu9fKVX3fny4Z4pRzuaqfJ3PBTMk2D9ur',6),
+        ('jefe-jefe','solana','JEFE','9RyCXca1aMBknh5285ybDgr9xxQhHxzXfghLq4HS2wUx',9),
+        ('rise-everrise-sol','solana','RISE','Pg4gfZvWZbzUbXSw4F4xoPdEg6WqiVt6M7LbMZ6ZraL',6),
+        ('chibi-chibi','solana','CHIBI','DieGFGgzKULVbc9jH57ek58yzDfbesUGAggvXHiaRd51',9),
+        ('pepe-pepe-by-matt-furie','solana','$PEPE','FVxWs1XeUEYECwqEDjor9WqCwRdW9TwkEubs5ACmuEYi',6),
+        ('woop-woop','solana','WOOP','A3HyGZqe451CBesNqieNPfJ4A9Mu332ui8ni6dobVSLB',5),
+        ('rope-rope-token','solana','ROPE','8PMHT4swUMtBzgHnh5U564N5sjPSiUz2cjEQzFnnP1Fo',9),
+        ('konke-konke','solana','KONKE','Ava2hJmCPhgZTbAaMDBM1M9A1P5VtvuVw5YLXV83ZTHQ',9),
+        ('apex-apexit-finance','solana','APEX','51tMb3zBKDiQhNwGqpgwbavaGH54mk8fXFzxTc1xnasg',9),
+        ('sfarm-solfarm','solana','SFARM','BjBzvw6VX7UJtrC7BaYLG1dHBiwrXP1T9j2YfDEdP4zU',6),
+        ('brix1-brix-gaming','solana','BRIX','BriXYhFVy3aXLiuVUfeG6ZMwHvifJLtxSuS5zgozDnAJ',9),
+        ('bot-starbots','solana','BOT','AkhdZGVbJXPuQZ53u2LrimCjkRP6ZyxG1SoM85T98eE1',8),
+        ('soopy-sopermen','solana','SOOPY','GjaH8mj1yjf7uDnZ4sKPn9my7cMQtZoXWPUeYyonMrEG',9),
+        ('ginger-gingers-have-no-sol','solana','GINGER','Ah1zCwmz1fm4A7Xt8WCHXtgPJshS14eCqvpmXD2xjCqs',9),
+        ('sail-sail','solana','SAIL','6kwTqmdQkJd8qRr9RjSnUX9XJ24RmJRSrU1rsragP97Y',6),
+        ('meals-meals-token','solana','MEALS','BLimDFGYnr6SNNWEGiXbe2FYSKgyHhjbVKMt47h5GezV',5),
+        ('vc-venture-coin','solana','VC','AebrVZPfSH7KPAxPwnuqTZB9QNepdktk7HSSY4SNj7BM',9),
+        ('chad-chad-on-solana','solana','CHAD','G7rwEgk8KgQ4RUTnMy2W2i7dRDq4hXHD4CSp9PSmSbRW',9),
+        ('mimo-million-monke','solana','MIMO','9TE7ebz1dsFo1uQ2T4oYAKSm39Y6fWuHrd6Uk6XaiD16',9),
+        ('net-internet','solana','NET','CcdBdzNBXc8GgKqy9DA8YnXmA1HFmaRTtdfF2uPH3Lkf',6),
+        ('pajeet-paje-etdev-company','solana','BOLLY','CPxE3TB2fjoxf3ip2DkepUMje8eCAucHHavS17g3VER1',6),
+        ('milk9-milk','solana','MILK','MLKmUCaj1dpBY881aFsrBwR9RUMoKic8SWT3u1q5Nkj',9),
+        ('eron-eron-moosk','solana','ERON','FThDUZ5y2R5qr5WnUPUN1nvB5CAWGFukLBsECdibFzYY',9),
+        ('chatni-chatni','solana','CHATNI','6CnDgbCpgeVhHWujctEwSLpVcGTZvGzR6eWh2VvB2nNo',9),
+        ('olananas-solananas','solana','OLANANAS','BUDDTuB9hQxYCxT2p45Q1yqx1667TYj2iT75tmM5BuCk',9),
+        ('shroom-shroom','solana','SHROOM','xyzR4s6H724bUq6q7MTqWxUnhi8LM5fiKKUq38h8M1P',9),
+        ('wipe-wipemyass','solana','WIPE','9ae76zqD3cgzR9gvf5Thc2NN3ACF7rqqnrLqxNzgcre6',9),
+        ('rug-rug','solana','RUG','BJRfDU8abGXmPiAj72kuc8AtXfxV8KFZAvdG3GTuB5Qc',8),
+        ('kuro-kurobi','solana','KURO','2Kc38rfQ49DFaKHQaWbijkE7fcymUMLY5guUiUsDmFfn',6),
+        ('mns-mansatrade','solana','MNS','MNSTRDvrtET2pesY6oBadd8qWXKVdcVyxWKf4CgUV3B',9),
```

Comment:
Inactive

Reply:
Was gonna ask about these. The reason I haven't removed or changed the inactive tokens is because the active status seems to be changing frequently. If you look at previous checks, some tokens that are now active, were inactive previously and vice versa.
previous check for reference - https://github.com/duneanalytics/spellbook/actions/runs/8509648889/job/23305640281?pr=5711

If you still want current inactives removed let me know, was just thinking maybe it might not be worth since others will likely pop up.